src/FormField.js

```diff
@@ -44,7 +44,20 @@ const getInputStyles = showLabel => {
 
 const noop = () => {}
 
-class InputField extends React.Component {
+const isFormElement = element => {
+  const typesOfElements = [Input, Select]
+  let isElement = false
```

Comment:
Might be simpler to handle this with an object/dict and do this like:
`const isFormElement = element => !!formElements[element]`

Reply:
Then `element` would need to be a `key` not an `object`. I could do a `map` but I'm not sure if that makes the code any better.